src/Microsoft.AspNetCore.Server.Kestrel/Http/SocketOutput.cs

```diff
@@ -535,7 +539,6 @@ private class WriteContext
 
             public int WriteStatus;
             public Exception WriteError;
-            public int ShutdownSendStatus;
```

Comment:
I don't really know. I think the perf would be the same. Brevity I guess. At least it's confined to a private class. I'm not going to change it for this PR.